spec/features/workflow/viewing_a_document_with_no_creator_spec.rb

```diff
@@ -24,6 +26,13 @@ def then_i_see_we_dont_know_who_created_the_document
     )
   end
 
+  def and_i_see_the_document_has_no_last_editor
+    expect(page).to have_content(
+      I18n.t("documents.index.search_results.last_edited_by",
+        user: I18n.t("documents.index.search_results.unknown_creator")),
```

Comment:
Could we create a separate translation for this, to avoid confusion between creator/editor?

Reply:
I think I'm going to go for Kevin's suggestion of an `unknown_user` here.